doc/getting_started/index.ipynb

```diff
@@ -441,23 +452,27 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "1d265d3b-6660-48b5-93e3-59b68ae82fa1",
-   "metadata": {},
+   "id": "c0bf904c-e06e-475a-bccb-2f58b5eb2681",
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "import cartopy.crs as crs\n",
     "\n",
     "proj = crs.Orthographic(-90, 30)\n",
     "air_ds.air.sel(time='2013-06-01 12:00').hvplot.quadmesh(\n",
-    "    'lon', 'lat', projection=proj, project=True,\n",
-    "    global_extent=True, cmap='viridis', coastline=True\n",
+    "    'lon', 'lat', projection=proj, global_extent=True,\n",
```

Comment:
Today we discussed starting the Geographic section with some simple example that doesn't leverage GeoViews and displays lat/lon data over web map tiles. Have you considered adding this?

Reply:
Sorry for the late reply, I was more thinking of a more simpler points plot (e.g. location of cities). But yes, the idea is not to use `geo=True`!

Reply:
Ok in fact I was thinking of not removing the original example but adding first a simpler example that doesn't require GeoViews. Basically, I don't want users to think they need *always* GeoViews to display geographic data, and show them when that's possible.